architecture.md

```diff
@@ -34,13 +27,7 @@ The entry point of the container is the Go UAST normalizer, which communicates
 internally with the language-specific code parser. This is how the Python driver
 looks like:
 
-```mermaid
-graph LR
-    Server-- stdin/stdout<br/>gRPC -->UAST
-    subgraph Container
-        UAST["UAST Normalizer<br/>(Go)"]-- stdin/stdout<br/>JSON -->AST["Code Parser<br/>(Python)"]
-    end
-```
+![Driver Architecture](./images/architecture-driver.png)
```

Comment:
What do the arrows in the pictures mean?

If they mean data communication, then maybe we should draw heads at both ends.

I think this also apply to the other picture.